src/command_modules/azure-cli-resource/tests/test_locks.py

```diff
@@ -0,0 +1,39 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+
+from azure.cli.testsdk import ScenarioTest, ResourceGroupPreparer
+
+
+class StorageAccountTests(ScenarioTest):
+    def test_list_locks(self):
+        locks_list = self.cmd('az lock list').get_output_in_json()
```

Comment:
This test might fail if the subscription has other locks. For example, this test will be included in our live test run in a subscription with unpredictable existing artifacts. 
I suggest just delete this test as the other new tests include `list` already.